### PR TITLE
[plugin.video.gdrive] 0.8.60

### PR DESCRIPTION
**Pull request by ddurdle**
### Description
Pushing the unofficial Google Drive video add-on (from 2014, ddurdle) to submission for the official repo.  3rd party authentication has been replaced local authentication.
### Checklist:

- [X ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.gdrive/resources/language/english/strings.xml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    <string id="30075">Enable sending crash logs to developer</string>
+    <string id="30076">Email address (optional)</string>
+    <string id="30077">Email address is for the developer to contact you at.</string>
+    <string id="30078">OS</string>
+    <string id="30079">XBMC version</string>
+    <string id="30080">Last identifier error #</string>
+
+    <string id="30095">Starred Files</string>
+    <string id="30096">Starred Folders</string>
+	<string id="30097">Starred Files & Folders</string>
```
Comment (ronie) [inline plugin.video.gdrive/resources/language/english/strings.xml]: ```&``` is not valid in xml files, please use ```&amp;``` instead

File: plugin.video.gdrive/resources/language/english/strings.xml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+	<string id="30098">Shared Files & Folders</string>
```
Comment (ronie) [inline plugin.video.gdrive/resources/language/english/strings.xml]: same here

File: plugin.video.gdrive/resources/lib/gPlayer.py [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+        except:
+            return
+
+    def PlayStream(self, url, item, seek, startPlayback=True, package=None):
+
+        self.currentURL = url
+        if startPlayback:
+            self.play(url, item)
+            if self.service.settings:
+                xbmc.log(self.service.addon.getAddonInfo('name') + ': Playback url ' + str(url), xbmc.LOGNOTICE)
```
Comment (ronie) [inline plugin.video.gdrive/resources/lib/gPlayer.py]: please use LOGDEBUG only in addons.

**Comment by ronie** [comment]
please remove icon.xcf

**Comment by ddurdle** [comment]
I'll make the changes and resubmit

**Pull request closed without merging**